Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN curl -L https://foundry.paradigm.xyz/ | bash && \
 # pnpm
 ENV PNPM_HOME="/pnpm"
 ENV PATH="${PATH}:${PNPM_HOME}"
-RUN npm install pnpm@8 --global && pnpm --version
+RUN npm install pnpm@9.1.1 --global && pnpm --version
```

Comment:
ideally this would install from the `packageManager` version so we don't have to redefine here but unclear how to do that